content/test/integration/service/garbage-collection/garbage-collection.spec.ts

```diff
@@ -243,6 +243,43 @@ describe('Integration - Garbage Collection', () => {
       expect(results?.gcStaleProfilesResult?.deletedHashes.size).toEqual(0)
       expect(results?.gcStaleProfilesResult?.deletedDeployments).toContain(p1DeploymentId)
     })
+
+    it('should not garbage collect default profiles even when they are old', async () => {
+      const timestamp = Date.now() - PROFILE_DURATION * 2
+      const defaultProfile = await buildDeployData(['default123'], {
```

Comment:
WDYT about setting const those string, maybe a beforeEach?
```suggestion
      const defaultProfile = await buildDeployData([PROFILE[1]], {
```